extensions/2.0/Vendor/EXT_texture_avif/README.md

```diff
@@ -0,0 +1,109 @@
+# EXT_texture_avif
+
+## Contributors
+
+- Omar Shehata, Cesium, [@OmarShehata](https://github.com/OmarShehata)
+- Pascal Massimino, Google, [@Skal65535](https://github.com/skal65535)
```

Comment:
The content of this section should not have been copy-pasted.

Reply:
I guess we are the authors?
I added myself, Don and you :)